Tests/Editor/Actions/StripProBuilderScripts.cs

```diff
@@ -0,0 +1,48 @@
+﻿using UObject = UnityEngine.Object;
+using System.Collections.Generic;
+using NUnit.Framework;
+using UnityEditor.ProBuilder;
+using UnityEngine;
+using UnityEngine.ProBuilder;
+using UnityEditor;
+
+public class StripProBuilderScripts
+{
+    bool m_OpenedWindow = false;
+    ProBuilderMesh m_cube;
+    GameObject m_cube_obj;
+
+    [Test]
+    public void Strip_ProBuilder_Scripts()
+    {
+        // make sure the ProBuilder window is open
+        if (ProBuilderEditor.instance == null)
+        {
+            ProBuilderEditor.MenuOpenWindow();
+            m_OpenedWindow = true;
+        }
+
+        UVEditor.MenuOpenUVEditor();
+
+        m_cube = ShapeGenerator.CreateShape(ShapeType.Cube);
+        m_cube_obj = m_cube.gameObject;
+        UnityEditor.ProBuilder.EditorUtility.InitObject(m_cube);
+
+        Assume.That(m_cube_obj.GetComponent<ProBuilderMesh>() != null);
+        Assume.That(m_cube_obj.GetComponent<PolyShape>() != null);
+
+        EditorApplication.ExecuteMenuItem("Tools/" + PreferenceKeys.pluginTitle + "/Actions/Strip All ProBuilder Scripts in Scene");
```

Comment:
Usually when writing tests it's best to keep the scope focused. In this case, we're testing the whole pipeline from `Menu Item` to the actual `Strip Script` function. So if this test fails it could be that the menu item function failed to find any `ProBuilderMesh` components in the scene, or that the stripping function itself failed to remove the proper components.

I'd suggest calling the `StripProBuilderScripts.Strip(ProBuilderMesh[] all)` function directly to avoid any ambiguity in the results, then maybe write an additional test that verifies the menu item does collect all meshes in the scene. I think for this PR however just testing the `Strip` function is fine.

Reply:
I now call DoStrip() on the object instead to avoid calls from the Editor (Strip had some UnityEditor code)